platformio.ini

```diff
@@ -587,7 +587,6 @@ build_flags_min =
 
 lib_deps_min =
   https://github.com/kosme/arduinoFFT#develop @ 1.9.2 ; used for USERMOD_AUDIOREACTIVE
-  https://github.com/netmindz/es8388arduino.git @ 0.0.1 ; used for USERMOD_AUDIOREACTIVE ES8388 source
```

Comment:
Not sure why this isn't showing in the final PR. 

Have we got a duplicate entry in the file?

Reply:
I think it was just out of sync